site/styles/_variables.scss

```diff
@@ -37,6 +37,7 @@ $primary:         #35a0a8 !default;
 $secondary:       #4cd3dd !default;
 $body-bg:         #0F0F0F !default;
 $body-color:      #fff !default;
+$body-bg-light:   lighten($body-bg, 4%) !default;
```

Comment:
How important is this, and what if the bg is white?

Reply:
@stajs This is needed because there is a new background colour for the mobile menu and dropdowns. With the old nav, you couldn't tell those elements apart from the background because they were the same colour.